query/src/test/java/org/infinispan/query/blackbox/ClusteredCacheWithLongIndexNameTest.java

```diff
@@ -64,32 +64,55 @@ public CacheMode getCacheMode() {
    }
 
    public void testAdditionOfNewNode() {
-      prepareData();
-
-      SearchManager sm = Search.getSearchManager(cache3);
-      QueryBuilder qb = sm.buildQueryBuilderForClass(VeryLongIndexNamedClass.class).get();
-      Query q = qb.keyword().wildcard().onField("name").matching("value*").createQuery();
-      CacheQuery<?> cq = sm.getQuery(q, VeryLongIndexNamedClass.class);
+      for (int i = 0; i < 100; i++) {
+         cache0.put("key" + i, new ClassWithLongIndexName("value" + i));
```

Comment:
Reading the JIRA, one entry should be enough -- although TBH this was a HibernateSearch bug, I don't think we need to keep a reproducer in our test suite.

Reply:
It was too cute to delete it...